docs/references/content-as-code.mdx

```diff
@@ -0,0 +1,47 @@
+# Content as code
+
+You can download your charts and dashboards as code, make changes, then upload your changes back to your Lightdash project using the CLI. Content as code is also useful for creating new Lightdash projects where you want the content to be copied from an existing project and added to your new project.
+
+## `lightdash download` downloads content as code
+
+From the [Lightdash CLI](/references/lightdash-cli), you can use the command `lightdash download` to download all of the charts and dashboards from your Lightdash project as code. All of the charts and dashboards will be written as .yml files to a `lightdash` directory inside of your dbt project. 
```

Comment:
> inside of your dbt project.

IT will create a directory wherever you're running your command. 

Eg: 

if I'm running this command inside my dbt directory (eg: /home/javi/dbt) then it will create a folder (/home/javi/dbt/lightdash). 
But if I'm running this command in /home/javi/documents it will create the folder in /home/javi/documents/lightdash

CLI does not know where your dbt folder is by default. and it doesn't need to know, download/upload works independently.

Reply:
ahhhh nice! makes sense